Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-volumegroup-snapshot.md

```diff
@@ -0,0 +1,107 @@
+---
+title: VolumeGroup Snapshots
+---
+
+## Prerequisites
+
+- Install the [snapshot controller and snapshot v1 CRD](https://github.com/kubernetes-csi/external-snapshotter/tree/master#usage),
+refer to VolumeGroupSnapshot documentation
+[here](https://github.com/kubernetes-csi/external-snapshotter/tree/master#volume-group-snapshot-support) for more details.
+
+- We also need a `VolumeGroupSnapshotClass` for volume group snapshot to work. The purpose of a `VolumeGroupSnapshotClass` is
+defined in [the kubernetes
+documentation](https://kubernetes.io/blog/2023/05/08/kubernetes-1-27-volume-group-snapshot-alpha/).
+In short, as the documentation describes it:
```

Comment:
I'd suggest deleting lines 14-18. I don't understand them. They can just naturally keep reading this doc to learn more.

Reply:
@travisn yes it can be deleted but we have similar one in the volume snapshot doc as well, This Doc is trying to align with the current format we are following.

Reply:
hmm, maybe it needs to be reworded, I just don't understand it.